box: [0.03, 0.03, 0.03]
contact_points: [0, 0.015, 0, 0, -0.015, 0, 0.08, 0, 0.015, 0.08, 0, -0.015]
contact_forces: [0, 0, 180, 0, 0, 0, 0, 0, 180, 0, 0, 0, 0, 0, 180, 0, 0, 0, 0, 0, 180, 0, 0, 0]
joints: [0,0,0,0,0,0,0,90,0,90,0,90,0,90]
mu: 1.5
f_i_max: 200
n_c: 4
n_q: 14